### PR TITLE
[IMP] Add binary mode in payment acquirer

### PR DESCRIPTION
**Pull request by maq-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/plugberry/mercadopago/pull/54).

File: payment_mercadopago/models/mercadopago_request.py [hunk 1]
```diff
@@ -163,6 +163,8 @@ def payment(self, tx, amount, capture=True, cvv_token=None):
         MercadoPago payment
         """
         _logger.info('Inicio tx con MP token %s' % cvv_token)
+        #import random
+        #return {'status':'pending', 'status_detail':'tests', 'id':int(random.random()*5000)}
```
Comment (bruno-zanotti) [inline payment_mercadopago/models/mercadopago_request.py]: @filoquin esto lo podemos borrar entiendo no?

**Comment by augusto-weiss** [comment]
@roboadhoc r+

**Comment by bruno-zanotti** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward.

**Comment by bruno-zanotti** [comment]
@roboadhoc override=ci/oba_without_tests

**Comment by roboadhoc** [comment]
I'm sorry, @Bruno-Zanotti: you are not allowed to override this status.

File: payment_mercadopago/models/mercadopago_request.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def payment(self, tx, amount, capture=True, cvv_token=None):
         MercadoPago payment
         """
         _logger.info('Inicio tx con MP token %s' % cvv_token)
-
```
Comment (bruno-zanotti) [inline payment_mercadopago/models/mercadopago_request.py:166]: eliminar cambio

File: payment_mercadopago/models/payment.py [hunk 3]
```diff
@@ -47,6 +47,7 @@ class PaymentAcquirerMercadoPago(models.Model):
         string='Capture method',
         default='deferred_capture'
     )
+    mercadopago_binary = fields.Boolean('Use binary mode', default=True)
```
Comment (bruno-zanotti) [inline payment_mercadopago/models/payment.py:50]: cambiar versión manifest

**Review by bruno-zanotti — CHANGES_REQUESTED** [review]

**Comment by bruno-zanotti** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Comment by fw-bot-adhoc** [comment]
@maq-adhoc @Bruno-Zanotti this pull request has forward-port PRs awaiting action (not merged or closed):
- plugberry/mercadopago#72

**Pull request closed without merging**